docs/tools/vdb_table/data/rockset.json

```diff
@@ -14,91 +14,91 @@
     "comment": ""
   },
   "license": {
-    "value": "",
+    "value": "Proprietary",
     "source_url": "",
     "comment": ""
   },
   "dev_languages": {
     "value": [
-      ""
+      "C++"
     ],
     "source_url": "",
     "comment": ""
   },
   "github_stars": 0,
-  "vector_launch_year": 0,
+  "vector_launch_year": 2023,
   "metadata_filter": {
-    "support": "",
+    "support": "full",
     "source_url": "",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "full",
```

Comment:
Please change this to none as Hybrid Search has a specific meaning in this tab le, which is different from what you explained.

Reply:
Hi @ankit1khare, the new repo you've linked to implements autocomplete functionality. Hybrid Search in this table refers to "ability to merge between ranked lists of keyword search and vector search results within the DB". Please revert this to "none"

Reply:
Hey @dhruv-anand-aintech, after all it's SQL we are talking about where we have built our own engine from scratch such that all standard SQL operations can be done on different data modalities. One can build their own weighting mechanism using SQL and get all of the analytical capabilities built into the query language (like linear combination, reciprocal rank fusion, etc.).
--

Reply:
Thanks for the explanation. It'll still be good to have an example notebook that does these types of fusion in Rockset, so that users are easily able to migrate to it from other DBs that do it under the hood.